doc/rfcs/0002-support-run-component-without-tiup.md

```diff
@@ -0,0 +1,23 @@
+# Support Run Component without TiUP
+
+## Summary
+
+Allow users to only use TiUP during installation and upgrade, without needing to invoke TiUP while using the component.
+
+## Motivation
+
+- Some users do not like the mandatory update checks and additional outputs by TiUP before invoking the components.
+- For programs that are not only distributed through TiUP, the existing usage patterns have led to documentation fragmentation.
```

Comment:
What programs does this refer to? sync-diff-inspector?

Reply:
For example, if an user follow https://docs.pingcap.com/tidbcloud/get-started-with-cli to install tidbcloud cli.He cannot use "ticloud" as doc, but have to use "tiup cloud"